menuflow/flow.py

```diff
@@ -49,3 +56,16 @@ def node(self, room: Room) -> Message | Input | HTTPRequest | None:
             return
 
         return node
+
+    def build_middleware(
+        self, data: Dict, type_class: HTTPMiddleware | None
+    ) -> HTTPMiddleware | None:
+        return type_class.deserialize(data)
```

Comment:
You must be reusing `build_node` and change its function name to `build` or `build_object`.